ddev/tests/conftest.py

```diff
@@ -83,7 +83,19 @@ def valid_integration(valid_integrations) -> str:
 
 
 @pytest.fixture(autouse=True)
-def config_file(tmp_path) -> ConfigFile:
+def config_file(tmp_path, monkeypatch) -> ConfigFile:
+    for env_var in (
+        'DD_GITHUB_USER',
+        'DD_GITHUB_TOKEN',
+        'DD_SITE',
+        'DD_DD_URL',
+        'DD_LOGS_CONFIG_DD_URL',
+        'DD_DD_URL',
+        'DD_API_KEY',
+        'DD_APP_KEY',
+    ):
+        monkeypatch.delenv(env_var, raising=False)
+
     path = Path(tmp_path, 'config.toml')
     os.environ[ConfigEnvVars.CONFIG] = str(path)
```

Comment:
Unrelated to this PR (so no need to change it here), but:

```suggestion
    monkeypatch.setenv(ConfigEnvVars.CONFIG) = str(path)
```

could save future trouble. Part of the point of using a fixture is isolation, we're making a permanent change to the global state here..